src/ol_dbt/models/marts/combined/_marts__combined__models.yml

```
@@ -105,6 +105,21 @@ models:
     description: number, the number of courses learner passed based on their emails
       so this counts as same person if they use the email to register on different
       platforms
+  - name: first_course_start_datetime
+    description: string, year and term from the users first course enrollment
+  - name: last_course_start_datetime
+    description: string, year and term from the users latest course enrollment
+  - name: number_of_courserun_certificates
+    description: int, number of course run enrollment certificates associated with
+      the user
+  - name: total_amount_paid_orders
+    description: int, sum of total orders paid associated with the user
+  - name: latest_income_usd
+    description: int, the users latest income in usd. Only populated for mitxonline.
+  - name: program_cred_ind
+    description: string, Set to true if the user has recieved a program credential
+  - name: dedp_program_cred_ind
+    description: string, Set to true if the user has recieved a dedp program credential
```

```suggestion
  - name: has_program_certificate
    description: boolean, Set to true if the user has received a program credential
  - name: has_dedp_program_certificate
    description: boolean, Set to true if the user has received a DEDP program credential
```

The name and data type could be clearer, and also some minor typos.